tests/acceptance/common.py

```diff
@@ -222,13 +222,16 @@ def determine_active_passive_part(bitbake_variables, conn):
         )
 
 
-def get_ssh_common_args():
-    return "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+def get_ssh_common_args(conn):
+    args = "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+    if "key_filename" in conn.connect_kwargs.keys():
+        args += " -i %s" % conn.connect_kwargs["key_filename"]
+    return args
 
 
 # Yocto build SSH is lacking SFTP, let's override and use regular SCP instead.
 def put_no_sftp(file, conn, remote="."):
-    cmd = "scp -C %s" % get_ssh_common_args()
+    cmd = "scp -C %s" % get_ssh_common_args(conn)
```

Comment:
I really wonder why we enable compression here, and everywhere within the acceptance tests. I would expect to disable the compression. is that on purpose?

Reply:
I don't understand this comment, I think you meant to comment somewhere else...

Reply:
Now I understand your comment :)

Good question! I don't know myself and it has been like this for very long time, so I don't dare to touch.

Reply:
I guess that for the QEMU testing you are correct, but the full framework was designed to also run tests in real devices. So if you have a device in Oslo and the test running in a GCP VM, then the transfer time would be longer that the compression, decompression, right?

Reply:
the question is how often we have a device in Oslo and test is running in GCP.

Reply:
0% in Oslo and 100% on GCP :wink:  This is very valid point, thanks for catching it. I will follow-up independently.